system/View/ViewDecoratorTrait.php

```diff
@@ -22,9 +21,7 @@ trait ViewDecoratorTrait
      */
     protected function decorateOutput(string $html): string
     {
-        $decorators = \config(ViewConfig::class)->decorators;
-
-        foreach ($decorators as $decorator) {
+        foreach ($this->config->decorators as $decorator) {
```

Comment:
This is a trait that is not marked internal and does not have a constructor or `$config` property. I'm surprised static analysis allowed this. Safer not to assume the property exists:
```suggestion
        $decorators = $this->config->decorators ?? \config(ViewConfig::class)->decorators;

        foreach ($decorators as $decorator) {
```

Reply:
Can we commit this and merge? This PR has been hanging out, approved, for a couple of months.